data.json

```diff
@@ -401,7 +401,7 @@
     "taishaneseRomanization": "[vak32 mi22]",
     "cantonese": "画眉",
     "mandarin": "画眉",
-    "english": "To paint one's  eyebrows",
+    "english": "To paint one’s eyebrows",
```

Comment:
I think the original apostrophe, `'`, was okay. You changed it to a [right single quotation mark](https://unicode-table.com/en/2019/).